bigbluebutton-html5/private/config/settings.yml

```diff
@@ -600,6 +600,8 @@ public:
     restoreOnUpdate: false
     oldMinimizeButton: false
     uploadEndpoint: '/bigbluebutton/presentation/upload'
+    uploadSizeMax: 30000000
```

Comment:
This is [partially] reverting https://github.com/bigbluebutton/bigbluebutton/pull/12100 where the agreement was to rely on the single configuration - bigbluebutton.properties
https://github.com/bigbluebutton/bigbluebutton/blob/v2.4.x-release/bigbluebutton-web/grails-app/conf/bigbluebutton.properties#L69-L75

Reply:
Could we reword it as [something like]:

```
# mirroredFromBBBCore are values controlled in bbb-web properties file. We include a copy here for notification purposes
mirroredFromBBBCore: 
  - uploadSizeMax: 30000000
  - uploadPagesMax: 200
```

Reply:
Yes, it seems more coherent that way.
I will change it soon.